### DISCUSSION TITLE
How can you create a new repository?

### DISCUSSION THREAD

**Discussion by skanda890** [community/community#75926] [Repositories] [answered]
### Select Topic Area

Question

### Body

How can you create a new repository in mobile platform?

**Comment by ghost** [comment, marked as answer]
Hello @skanda890 

In home page you should be able to see this green button (button with the text "new")
![image](https://github.com/community/community/assets/101295551/351ac8c2-a2d9-4c27-a35a-f2fa6823897f)

once you click that button you will be redirected to repository creation page where you should type general config of your repo:

- repository name - repository name should be unic for each user (you will not be able to create 2 repository with exact same name)
- repository visibility - by default it is set to public (everyone will be able to see your repo). if you plan to work on something you do not want others to see than you should defenitelly select "private"  option there.
- you can add README, License and .gitignore files (each has its own specific usecase, you can see it [there](https://github.com/vanatteveldt/github-workshop/blob/main/tutorials/readme_license_gitignore.md) )

after that you should click "create repository" and it will be created.

There is one last part (to initialize your repo). you have 3 options there:
- you can create brand new repo from command line tool (terminal). to do so copy this option out of 3: 
![image](https://github.com/community/community/assets/101295551/8a6f0ae0-70e6-4a37-a097-007b0fe2e967)
navigate to your project root folder from command line. paste coppied code in terminal and execute it. than refresh your github page and you will see fully functional github repo with README file added in it.

- second option is to push existing repository from command line. to do so navigate to existing repo from terminal. copy this code in github page: 
![image](https://github.com/community/community/assets/101295551/3acbaefd-6f37-495d-85cc-078cc841ce22)
 and execute it from terminal. then refresh github page and you will see locally existing repo is pushed to github.

- Third option is to import code from other repository. to do so click on thi button and follow steps: 
![image](https://github.com/community/community/assets/101295551/dd61b037-4180-406d-aa9c-e43bd8a1bd09)

I suggest to check more information about how to create github repo [there](https://docs.github.com/en/get-started/quickstart/create-a-repo)

Hope this helps
